comparisons/validate.C

```diff
@@ -1641,93 +1658,32 @@ void validateEvents(TString step, TString file, TString refFile, TString r="RECO
 
   std::cout<<"Start making plots for Events with "<<Nnew<<" events and refEvents with "<<Nref<<" events ==> check  "<<Nmax<<std::endl;
 
-  if (!stepContainsNU(step, "hlt")){
-
+  if (stepContainsNU(step, "NANO")){
```

Comment:
this part of the change would not work.
The driver shell script assigns everything to step `all` and it's up to `validateEvents` to figure out what to plot (in most cases just try everything).

It looks like the EDM files have some "standard" branches like `EventAuxiliary` or `EventProductProvenance` in the `Events` tree. Perhaps these could be used to guess the file content.

A more controllable alternative is to extend the input map file format to have one more field for the type of a file (nano, reco, or even mini). 
The current format of the map file `shortName fullName.root processName comments`.

Reply:
I thought, without checking, that the third was the `step` that was passed to the script indeed.
I’ll switch back to the initial if statement